### ISSUE TITLE
`plot()` outputs require white background (or white lines?) for the dark mode website

### ISSUE THREAD

**Issue by avallecam** [r-lib/pkgdown#2729]
Currently, the `plot()` outputs are lost in the background when the dark mode is used.

Possibly the standard suggested action to do would be to try to keep the default white background.

reference link: https://pkgdown.r-lib.org/articles/test/rendering.html

![image](https://github.com/user-attachments/assets/b3cb6432-9866-42a8-83ed-8dc8afcaca70)

**Comment by jayhesselberth** [comment]
I believe this is expected behavior as the default background color for base plots is transparent.

I set the background color to white in #2730 and the preview looks find in dark mode.

**Comment by avallecam** [comment]
I agree with the approach to let users set up the parameter. Thanks for adding that test to the vignette!

**Comment by hadley** [comment]
Hmmm, are you sure we shouldn't set the background colour with css?

**Comment by jayhesselberth** [comment]
Added the CSS approach to #2730, works fine so long as we explicitly set the logo background to transparent.

Or maybe we should only set background to white if dark-mode is active?

**Comment by jayhesselberth** [comment]
Are we not adding a CSS class to plot images? Would make it easier to target their backgrounds specifically.

**Comment by Bisaloo** [comment]
Linking to https://github.com/r-lib/pkgdown/issues/2854. I expect it should also solve the issue here.
